src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -918,6 +934,46 @@ map_injective f.injective u v
 
 end path
 
+/-! ## Deleting edges -/
+
+namespace walk
+variables {G}
+
+/-- Given a walk that avoids a set of edges, produce a walk in the graph
+with those edges deleted. -/
+def to_delete_edges (s : set (sym2 V)) :
```

Comment:
I would suggest
```suggestion
protected def delete_edges (s : set (sym2 V)) :
```
to exactly match the name of the underlying construction.

Reply:
The reason I chose `to_delete_edges` rather than `delete_edges` was that `delete_edges` makes it sound like edges are being deleted from the path itself, and this is a map from walks on `G` to walks on `G.delete_edges`.